lang/python/python3/Makefile

```diff
@@ -282,6 +285,9 @@ define Py3Package/python3-base/install
 	$(LN) python$(PYTHON3_VERSION) $(1)/usr/bin/python3
 	$(LN) python$(PYTHON3_VERSION) $(1)/usr/bin/python
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libpython$(PYTHON3_VERSION).so* $(1)/usr/lib/
+  # This depends on being called before filespec is processed
+	$(SED) 's|$(TARGET_AR)|ar|g;s|$(TARGET_CROSS)readelf|readelf|g;s|$(TARGET_CC)|gcc|g;s|$(TARGET_CXX)|g++|g' \
+		$(PKG_INSTALL_DIR)/usr/lib/python$(PYTHON3_VERSION)/_sysconfigdata.py
```

Comment:
i'm a little vague on the details here; it's been a while since i looked over `_sysconfigdata.py`
but, would it be easier to just fixup the prefix?
or, does the prefix differ in these tools?
i'm also assuming the prefix is something like `$(STAGING_DIR)` or something like that

tbh, i have no idea which approach is more future-proof  ¯\_(ツ)_/¯

Reply:
The prefix is `$(TARGET_CROSS)`, e.g. for armvirt-64 it’s `aarch64-openwrt-linux-musl-`. When using an external toolchain, this value will be different.

On target, gcc and g++ are available with and without this prefix, i.e. `gcc` is a [symlink](https://github.com/openwrt/packages/blob/master/devel/gcc/Makefile#L194) to `aarch64-openwrt-linux-musl-gcc`, but [ar](https://github.com/openwrt/openwrt/blob/master/package/devel/binutils/Makefile#L112) and [readelf](https://github.com/openwrt/openwrt/blob/master/package/devel/binutils/Makefile#L117) are only available without this prefix.

I think the simplest approach (for now) is to remove the prefix completely :joy:

Reply:
ok